src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.css

```diff
@@ -1,60 +1,184 @@
 .summary-item-field a {
-    white-space: nowrap;
+    white-space: nowrap
 }
 
 .summary-item-field-heading-first {
-    font-size: 1rem;
+    font-size: 1rem
 }
 
 .summary-item-field-heading {
-    font-size: 1rem;
+    font-size: 1rem
 }
 
-@media only screen and (max-width: 760px),
-(min-device-width: 768px) and (max-device-width: 1024px) {
+.create-opportunity-link {
+    padding-bottom: 0px
+}
+
+.create-opportunity-link p {
+    margin-bottom: 1rem
+}
+
+section.live-brief-container {
+    margin-top: 3.5rem
+}
+
+.dashboard-main {
+    width: 74.48861%
+}
+
+@media (max-width: 767px) {
+    .dashboard-main {
+        width: 100%
+    }
+
     .table-row {
         position: absolute;
         top: -9999px;
-        left: -9999px;
+        left: -9999px
     }
 
     .closed-item-row {
-        display: block;
+        display: block
     }
 
     .closed-item-field-title {
         display: inline-block;
         width: 33.3% !important;
-        border: hidden !important;
+        border: hidden !important
     }
 
     .closed-item-field-date {
         display: inline-block;
         width: 33.3% !important;
-        border: hidden !important;
+        border: hidden !important
     }
 
     .closed-item-field-actions {
         display: inline-block;
-        width: 33.3% !important;
+        width: 33.3% !important
     }
 
-    .closed-item-field-actions a{
-        white-space: nowrap;
+    .closed-item-field-actions a {
+        white-space: nowrap
     }
 
     .item-field {
         display: inline-block;
         width: 33.3% !important;
-        border: hidden !important;
+        border: hidden !important
+    }
+
+    .item-field a {
+        overflow-wrap: break-word;
+    }
+
+    table.brief-table td {
+        padding: 1em 1.2em
     }
 
-    .summary-item-field-heading {
-        vertical-align: bottom !important;
+    table.brief-table thead tr th.middle-header {
+        padding-right: 20px;
+        text-align: left;
+        width: 50%
     }
 
-    .summary-item-field-heading-first {
-        vertical-align: bottom !important;
+    table.brief-table thead tr th.right-header {
+        width: 10%
     }
 }
 
+table.brief-table {
```

Comment:
It replaces styling from "content-tables" from a uikit style sheet. I don't think there's any doubling up within this file though.